moveit_ros/moveit_servo/launch/servo_example.launch.py

```diff
@@ -19,6 +19,7 @@ def generate_launch_description():
         .yaml("config/panda_simulated_config.yaml")
         .to_dict()
     )
+    servo_params = {"moveit_servo": servo_params}
```

Comment:
Can you do this in one line without overwriting the `servo_params` variable like you did in the pose tracking launch file?

Reply:
ah yes, will do.